File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/TaggedMachineBase.scala

package org.openapitools.client.model


case class TaggedMachineBase (
    /* The ID of the Machine being tagged. */
    _id: Long
)
object TaggedMachineBase {
    def toStringBody(var_id: Object) =
        s"""
        | {
        | "id":$var_id
        | }
        """.stripMargin
}

File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/TaggedMachineBase.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param id The ID of the Machine being tagged.
 */
data class TaggedMachineBase (

        @get:NotNull 
        @JsonProperty("id") val id: Long
) {

}

